docs/extensions-plugins/cli/commands.mdx

```diff
@@ -73,12 +73,22 @@ pieces list models
 ```
 
 ### `open`
```

Comment:
Can we here say
`#open (deprecated)`

Reply:
addressed in the commit [36435e3](https://github.com/pieces-app/documentation/pull/538/commits/36435e388b95e0e9ee157bcfc3691c6d2774ed2a) commit .  also just to be sure , I think you meant ```#### open (deprecated)```  instead of ```###open (deprecated)``` . if not , then I'll make the change exactly as suggested .